testdata/rust/rust-stable/plan.json

```diff
@@ -1,15 +1,18 @@
 {
   "install_stage": {
-    "command": "",
+    "command": "rustup default stable",
     "input_files": [
-      "."
+      "Cargo.toml"
     ]
   },
   "build_stage": {
-    "command": ""
+    "command": "cargo build",
+    "input_files": [
+      "."
+    ]
   },
   "start_stage": {
-    "command": "",
+    "command": "rustup default stable \u0026\u0026 cargo run \u0026\u0026 rustc --version",
```

Comment:
Why do we need `\u0026`. Is that what the go json marshaller does? Can we make it output `&` instead?

Reply:
its from json marshaller. I'll see if there's a way to avoid it...